README.md

```diff
@@ -69,13 +67,10 @@ The translations of these metrics are available at [chaoss/translations](https:/
 
 ### Amazing CHAOSS Project Contributors
 
-We greatly appreciate our contributors at CHAOSS and look forward to your joining us as well. All CHAOSS Project contributors are listed [here](https://chaoss.community/metrics/#user-content-chaoss-contributors-include)
-
-Are you eligible to be on this list? You are if you helped in any capacity, for example: Filed an issue. Created a Pull Request. Gave feedback on our work. The team will try to update this list regularly, but please open an issue or post
-on the mailing list if we've missed anyone.
+We'd like to thank everyone who has contributed to the CHAOSS project and our different working groups. We know that there are many different [types of contributions](https://chaoss.community/kb/metric-types-of-contributions/) and would like to recognize everyone's amazing work in these different areas. Thank You!!
 
 ## License
 
-The documents in this repository are released under the MIT License. See [LICENSE](LICENSE) file.
+See [LICENSE](LICENSE) file.
 
-Copyright © 2019-2022 - CHAOSS, a Linux Foundation Project
+Copyright © 2023 - CHAOSS
```

Comment:
LF Guidance is to leave the year off: https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

We should probably change ours to be one of these for all of our projects:
* Copyright The XYZ Authors.
* Copyright The XYZ Contributors.
* Copyright Contributors to the XYZ project.